agents/src/llm/llm.ts

```diff
@@ -3,16 +3,12 @@
 // SPDX-License-Identifier: Apache-2.0
 import { AsyncIterableQueue } from '../utils.js';
 import type { ChatContext, ChatRole } from './chat_context.js';
-import type {
-  CallableFunctionResult,
-  DeferredFunction,
-  FunctionContext,
-} from './function_context.js';
+import type { DeferredFunction, FunctionContext } from './function_context.js';
 
 export interface ChoiceDelta {
   role: ChatRole;
   content?: string;
-  toolCalls?: FunctionContext;
+  toolCalls?: DeferredFunction[];
```

Comment:
We should call it `FunctionCallInfo` to match with Python?

Reply:
see slack message, i think it might be better to name it differently because it's so fundamentally different than the python impl and it would be more confusing to name it similarly

Reply:
But what happens if I want to simulate a conversation? 
For example I want to load a serialized ChatContext from a database and it contains a CallInfo

Reply:
hmm yeah, that's fair. i'll rename it to functioncallinfo